tools/motus/motus_profiler.xml

```diff
@@ -0,0 +1,220 @@
+    ]]></command>
+    <inputs>
+        <section name="inputs" title="Inputs" expanded="true">
+            <conditional name="input_type">
+                <param name="selector" type="select" label="Input file(s)">
+                    <option value="reads" selected="True">fastq,fastq.gz file(s)</option>
+                    <option value="SAM_BAM_input">SAM/BAM input file(s) generated by motus map_tax</option>
+                    <option value="mgc_read_count">mgc_read_count file(s) generated by motus cal-mgc</option>
+                </param>
+                <when value="reads">
+                    <conditional name="reads_type">
+                        <param name="selector" type="select" format="fastq,fastq.gz" label="Choose input file(s)">
+                            <option value="paired" selected="true">fastq,fastq.gz Forward and Reverse Reads</option>
+                            <option value="single" selected="true">fastq,fastq.gz Unpaired reads</option>
+                        </param>
+                        <when value="paired">
+                            <param name="forward_reads" type="data" format="fastq,fastq.gz" label="Forward Reads" help="-f Input file(s) for reads in forward orientation, fastq(.gz)-formatted"/>
+                            <param name="reverse_reads" type="data" format="fastq,fastq.gz" label="Reverse Reads" help="-r Input file(s) for reads in reverse orientation, fastq(.gz)-formatted"/>
+                        </when>
+                        <when value="single">
+                            <param name="unpaired_reads" type="data" format="fastq,fastq.gz" label="Unpaired reads" help="-s input file(s) for unpaired reads, fastq(.gz)-formatted"/>
+                        </when>
+                    </conditional>
+                </when>
+                <when value="SAM_BAM_input">
+                    <param name="SAM_BAM_input" type="data" format="sam,bam" label="motus map_tax SAM or BAM file" help="-i SAM or BAM input file(s) generated by motus map_tax"/>
+                </when>
+                <when value="mgc_read_count">
+                    <param name="mgc_count" type="data" format="mgc" label="mgc read count file)(s)" help="-m mgc reads count file generated by motus calc_mgc"/>
+                </when>
+            </conditional>
+            <param name="sample_name" type="text" label="Sample name" help="sample name can contain letters and/or numbers" default_value="unnamed sample"/>
+            <conditional name="db">
+                <param name="db_source" type="select" label="mOTUs Database">
+                    <option value="cached">Use a pre-installed database</option>
+                    <option value="history">Use a database from history</option>
+                </param>
+                <when value="cached">
+                    <param name="db_cached" type="select" label="A pre-installed mOTUs databese" help="no pre-installed mOTUs databases are avalable yet">
```

Comment:
This option needs a test. @paulzierep probably need to help here a bit.

Also as a small rule, try to add tests so that most if not all conditional options are covered.

Reply:
yes, I am working on a DM atm, will make a PR here, to add the missing parts for the cached choice ones I tested the DM

Reply:
So, they will provide a small DB (still soonish), until then I will set up the DM and fix the boilerplate, but work with 1 test ...